db-service/package.json

```diff
@@ -25,6 +25,9 @@
     "node": ">=16",
     "npm": ">=8"
   },
+  "scripts": {
+    "test": "jest --silent"
```

Comment:
```suggestion
    "test": "jest --silent --colors"
```

Reply:
I don't like the pattern to always define `--colors` in every `jest` call. My understanding is that this is only to have color in the Github action. Therefor I have added the environment variable `FORCE_COLOR` which will enable it explicitly only in the Github action. Assuming that it will behave correctly in all other scenarios.